DOCUMENTATION.md

```diff
@@ -336,19 +336,35 @@ To find the template ID:
 
 #### personalisation (required)
 
-The personalisation argument always contains the following required parameters for the letter recipient’s address:
+The personalisation argument always contains the following parameters for the letter recipient’s address:
 
 - `address_line_1`
 - `address_line_2`
-- `postcode` (this needs to be a real UK postcode)
+– `address_line_3` 
+– `address_line_4` 
+– `address_line_5` 
+– `address_line_6`
+– `address_line_7`
+
+The address must have at least 3 lines.
+
+The last line needs to be a real UK postcode or the name of a country outside the UK.
+
+Notify checks for international addresses and will automatically charge you the correct postage.
+
+The `postcode` personalisation argument has been replaced. If your template still uses `postcode`, Notify will treat it as the last line of the address.
```

Comment:
this one still feels a little weird (cos its explaining such a weird legacy edge casey thing, i suppose), but i think this is fine for now